app/screens/get-started-screen/get-started-screen.tsx

```diff
@@ -109,11 +105,49 @@ export const GetStartedScreen: React.FC<Props> = ({ navigation }) => {
           />
         ) : (
           <GaloySecondaryButton
-            title={LL.GetStartedScreen.exploreWalletInstead()}
+            title={LL.GetStartedScreen.exploreWallet()}
             onPress={handleExploreWallet}
           />
         )}
+        <View style={styles.container}>
+          <Text style={styles.text}>{LL.GetStartedScreen.loginBackWith()} </Text>
+          <TouchableOpacity activeOpacity={0.5} onPress={handleCreateAccount}>
+            <Text style={styles.buttonText}>{LL.common.phone()}</Text>
+          </TouchableOpacity>
+          <Text style={styles.text}> {LL.common.or()} </Text>
+          <TouchableOpacity activeOpacity={0.5} onPress={handleLoginWithEmail}>
+            <Text style={styles.buttonText}>{LL.common.email()}</Text>
+          </TouchableOpacity>
+        </View>
       </View>
     </Screen>
   )
 }
+
+const useStyles = makeStyles(() => ({
+  bottom: {
+    flex: 1,
+    paddingHorizontal: 24,
+    justifyContent: "flex-end",
+    marginBottom: 36,
+  },
+
+  buttonContainer: {
+    marginVertical: 6,
+  },
+
+  logoContainer: { width: "100%", height: "50%", marginTop: 50 },
+
+  container: {
+    marginTop: 24,
+    justifyContent: "center",
+    flexDirection: "row",
+  },
+  text: {
+    fontSize: 18,
```

Comment:
I would not set fontSize in the text style but instead use one of the text variants by adding the prop `type="p2"`